src/routes/event-team-comments.tsx

```diff
@@ -46,8 +50,12 @@ const EventTeamComments = ({ eventKey, teamNum }: Props) => {
 
   return (
     <Page
-      name={`Comments: ${teamNum} @ ${eventInfo ? eventInfo.name : eventKey}`}
-      back={`/events/${eventKey}/teams/${teamNum}`}
+      name={
+        <>
+          {`Comments: ${teamNum} @ `}
+          <a href={`/events/${eventKey}`}> {eventName} </a>
```

Comment:
We do not need these extra spaces
```suggestion
          <a href={`/events/${eventKey}`}>{eventName}</a>
```